.travis/cpp_style_comment_linter.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+# Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Copyright should be 2019

Reply:
I just saw that too. the change has been pushed